superset-frontend/src/common/components/Tabs.tsx

```diff
@@ -59,4 +85,48 @@ const Tabs = Object.assign(StyledTabs, {
   TabPane: StyledTabPane,
 });
 
+Tabs.defaultProps = {
+  fullWidth: true,
+};
+
+const StyledEditableTabs = styled(StyledTabs)`
```

Comment:
We should add this to the Storybook entry for tabs (as another Story, if that makes sense)

Reply:
Nevermind, you did that!!!! 🎉